hack/test/basic-integration.sh

```diff
@@ -3,93 +3,32 @@
 set -eou pipefail
 
 
-export KUBERNETES_VERSION=v1.17.0
-export TALOS_IMG="docker.io/autonomy/talos:${TAG}"
-export TMP="/tmp/e2e"
-export TALOSCONFIG="${TMP}/talosconfig"
-export KUBECONFIG="${TMP}/kubeconfig"
-export TIMEOUT=300
-export OSCTL="${PWD}/${ARTIFACTS}/osctl-linux-amd64"
-export INTEGRATIONTEST="${PWD}/bin/integration-test"
+TALOS_IMG="docker.io/autonomy/talos:${TAG}"
+OSCTL="${PWD}/${ARTIFACTS}/osctl-linux-amd64"
+INTEGRATIONTEST="${PWD}/bin/integration-test"
+
+
+case "${CI:-false}" in
```

Comment:
I don't see where CI is getting set. Does Drone set it?

Reply:
yep, it's one of the Drone environment variables